lib/namespaces.js

```diff
@@ -11,7 +11,10 @@ const DEFAULT_RESOURCES = [
   { name: 'replicationcontrollers', Constructor: ReplicationControllers },
   { name: 'pods', Constructor: Pods }
 ].concat([
+  'clusterroles',
```

Comment:
Could we keep these strings in one place? we use them for `resourceAliases` as well as keep them in the constructors of the various `ApiGroups`. We should centralize these strings so updating them is not painful.